components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/ApisApiServiceImpl.java

```diff
@@ -265,6 +265,46 @@ public Response addCommentToAPI(String apiId, PostRequestBodyDTO postRequestBody
         return null;
     }
 
+    /**
+     * Retrieves the async api specification document of an API
+     *
+     * @param apiId API identifier
```

Comment:
Shall we format the Javadoc comments?

Reply:
Sure. Will update with another PR since this is already merged